glib/src/value.rs

```diff
@@ -822,6 +836,13 @@ impl ToValue for String {
     }
 }
 
+impl From<String> for Value {
```

Comment:
Not also `From<&str>` and `From<&[&str]>`?

Reply:
Those get handled by the blanket `From<&T>` impl